docs/faq/env_var.md

```diff
@@ -110,9 +114,12 @@ When USE_PROFILER is enabled in Makefile or CMake, the following environments ca
 ## Other Environment Variables
 
 * MXNET_CUDNN_AUTOTUNE_DEFAULT
-  - Values: 0(false) or 1(true) ```(default=1)```
-  - The default value of cudnn auto tunning for convolution layers.
+  - Values: 0, 1, or 2 ```(default=1)```
+  - The default value of cudnn auto tuning for convolution layers.
   - Auto tuning is turned off by default. For benchmarking, set this to 1 to turn it on by default.
```

Comment:
Here it states the default is turned off, but above statement states that default is 1 ?

Reply:
Weird. Corrected